op-e2e/actions/l2_sequencer.go

```diff
@@ -28,6 +28,18 @@ func (m *MockL1OriginSelector) FindL1Origin(ctx context.Context, l2Head eth.L2Bl
 	return m.actual.FindL1Origin(ctx, l2Head)
 }
 
+// emptyL1BlobsFetcher is a no-op blobs provider .The actions test batcher currently only supports using calldata.
```

Comment:
"provider .The" -> "provider. The"

But also, could you rephrase this comment somewhat? I think it means that this struct is used by e2e tests only, in order to confirm that blobs aren't used pre-dencun, but I'm not sure.